spec/Node.ts

```diff
@@ -0,0 +1,21 @@
+export const enum NodeKind {
+  Chain = "Chain",
+  Metadata = "Metadata",
+  Pallet = "Pallet",
+  Block = "Block",
+  StorageItem = "StorageItem",
+  StorageMap = "StorageMap",
+  StorageMapValue = "StorageMapValue",
+  StorageMapKeysPage = "StorageMapKeysPage",
+  Access = "Access",
+  ExtrinsicFactory = "ExtrinsicFactory",
+  Extrinsic = "Extrinsic",
+  SignedExtrinsic = "SignedExtrinsic",
+  Event = "Event",
+  Read = "Read",
+  Subscribe = "Subscribe",
+}
```

Comment:
I thought I had rid this land of these pests!

Reply:
Whoops. String literals from here on out. Fingers crossed!